lib/helpers/pack_hoodie.js

```diff
@@ -105,20 +106,21 @@ module.exports = function (env_config) {
   // Create the `bundleConcat` that we'll use to create the `browserify` bundle.
   fs.writeFileSync(bundleConcat, bundleTmpl + concatPlugins(plugins));
 
-  var b = browserify(bundleConcat);
-
-  return b.bundle({
+  var b = browserify(bundleConcat, {
     standalone: 'Hoodie',
     external: 'jquery'
-  }, function (err, bundle) {
+  });
+
+  b.bundle(function (err, bundle) {
     if (err) { return console.error(err); }
     // Store successful bundle in cache.
     cache = bundle;
     // Clean up! Remove dynamically generated file with concatenated scripts
     // once we are done building the bundle.
-    fs.unlink(bundleConcat, function (err) {
+    fs.unlinkSync(bundleConcat, function (err) {
```

Comment:
[`fs.unlikSync`](https://iojs.org/api/fs.html#fs_fs_unlinksync_path) is the synchronous version of [`fs.unlink`](https://iojs.org/api/fs.html#fs_fs_unlink_path_callback). That means the second argument (the anonymous function) will never be called.